packages/common/dist/schemas/apiVersions/0.1.0-0.1.0/openApi.json

```diff
@@ -10074,6 +10074,7 @@
                   "attributes.collectingEventDistrict",
                   "attributes.collectingEventEndDate",
                   "attributes.collectingEventLocality",
+                  "attributes.collectingEventInterpretedLocality",
```

Comment:
why is this and taxonomyRank in this PR? it should have been in an earlier PR, right?